app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +23,9 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.service.quicksettings.action.QS_TILE_PREFERENCES"/>
```

Comment:
What does it do?

Reply:
Adds the ability to change the tile icon?

Reply:
No, this is just for intercepting a long-press on the Tile.
By default, it would open the application information screen.
If you use this intent filter, the MainActivity will be launched, allowing you to update the proxy if you long-press the tile.